tests/unit-core/test-get-own-property.c

```diff
@@ -0,0 +1,174 @@
+ /* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript.h"
+
+#include "test-common.h"
+
+static jerry_value_t
+create_object (const char *source_p) /**< source script */
+{
+  jerry_value_t result = jerry_eval ((const jerry_char_t *) source_p, strlen (source_p), 0);
+  TEST_ASSERT (jerry_value_is_object (result));
+  return result;
+} /* create_object */
+
+static void
+compare_string (jerry_value_t value, /**< value to compare */
+                const char *string_p) /**< expected value */
+{
+  jerry_char_t string_buffer[64];
+
+  TEST_ASSERT (jerry_value_is_string (value));
+
+  size_t size = strlen (string_p);
+  TEST_ASSERT (size <= sizeof (string_buffer));
+  TEST_ASSERT (size == jerry_get_string_size (value));
```

Comment:
As we are accessing the string as an utf8 string, should we also check the utf8 size here (`jerry_get_utf8_string_size`)?

Reply:
using the utf8 was a mistake